deps/v8/src/compiler/functional-list.h

```diff
@@ -106,6 +107,14 @@ class FunctionalList {
     }
     bool operator!=(const iterator& other) const { return !(*this == other); }
 
+    // Implemented so that std::find and friends can use std::iterator_traits
+    // for this iterator type.
+    typedef std::forward_iterator_tag iterator_category;
+    typedef ptrdiff_t difference_type;
+    typedef A value_type;
+    typedef A* pointer;
+    typedef A& reference;
```

Comment:
I think you may have added these changes by accident? If you do want them to land in master, I’d recomment opening a separate backport PR, in the format of e.g. https://github.com/nodejs/node/pull/31354

Reply:
Yes, this was require patch to compile node in GCC. It is not part of the feature. I excluded that now.